### PR TITLE
chore(CI): add CircleCI config

### PR DESCRIPTION
**Pull request by Thomaash**
Installs, tests, builds and releases (semantic-release) if necessary.
Linting is commented out at the moment (too many failures). It should be uncommented later when the code base is compliant.

Addresses #1.

PS @mojoaxel: I can't set this (or any other repo) up on CircleCI. It says: “You must be an admin to add a project on CircleCI”.

### CODE REVIEW

**Comment by mojoaxel** [comment]
> I can't set this (or any other repo) up on CircleCI. It says: “You must be an admin to add a project on CircleCI”.

@Thomaash You are now also a admin of this repository. Please be very responsible! (One time a contributor deleted the whole almende/vis repository from github, by mistake.)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vis-network",
-  "version": "5.1.1",
+  "version": "0.0.0-no-version",
```
Comment (mojoaxel) [inline package.json]: why this?

File: .releaserc.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+branch: master
+plugins:
+  - '@semantic-release/commit-analyzer'
```
Comment (mojoaxel) [inline .releaserc.yml]: do we need this packages as dependency in package.json?

**Comment by Thomaash** [comment]
Thanks, I'll be very careful.

File: .releaserc.yml (hunk 2 above)
Comment (Thomaash) [reply .releaserc.yml]: Nope, semantic-release depends on these.

File: package.json (hunk 1 above)
Comment (Thomaash) [reply package.json]: We will release through semantic-release which doesn't update this. In other words we would have 5.1.1 forever.

**Pull request merged**